### PR TITLE
test: Add viz storybook examples

### PR DESCRIPTION
**Pull request by mirisb**
New examples in storybook that mimics how the component is used inside external applications to simulate layout bugs that are observed there.

New example uses helper files that simulates the environment where the component is rendered.

Also - this specific example is wrapped in React.StrictMode component to identify additional issues.

image

### CODE REVIEW

File: typescript/packages/well-log-viewer/src/Storybook/helpers/mui-components.tsx [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+/**
+ * Remove spaces and set to lower case.
+ * Note that we cannot use replaceAll function
+ * below until we move to ES2021.
+ */
+const buildId = (id: string): string => {
+    return id.replace(/ /g, "").toLowerCase();
+};
+
+export type TooltipProps = Pick<
```
Comment (w1nklr) [inline typescript/packages/well-log-viewer/src/Storybook/helpers/mui-components.tsx:24]: Can you narrow that down ?
For example, if the tooltip is not part of the pb, you can remove it (it's like 25% of this file).

**Review by w1nklr — CHANGES_REQUESTED** [review]

File: typescript/packages/well-log-viewer/src/Storybook/helpers/mui-components.tsx
[the remark this answers is not in the document]
Comment (mirisb) [reply typescript/packages/well-log-viewer/src/Storybook/helpers/mui-components.tsx:1]: done

File: typescript/packages/well-log-viewer/src/Storybook/ExternalLayout.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                >
+                    <SyncLogViewer {...{ ...args, id: "c1" }} />
+                </div>
+            </div>
+        </Grid>
+    );
+};
+
+// storybook page
+const meta: Meta<typeof RTCWellLogViewer> = {
```
[the remark this answers is not in the document]
Comment (mirisb) [reply typescript/packages/well-log-viewer/src/Storybook/ExternalLayout.stories.tsx:30]: not possible as the component the meta is based on is defined above

File: typescript/packages/well-log-viewer/src/Storybook/ExternalLayout.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+import { Grid } from "@mui/material";
+//import type { GridDirection } from "@mui/material";
+
+import SyncLogViewer from "../SyncLogViewer";
+
+import { tabDecorator } from "./helpers/MuiComponentsTabHelper";
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const args = require("../../../../../example-data/facies3wells.json");
```
[the remark this answers is not in the document]
Comment (mirisb) [reply typescript/packages/well-log-viewer/src/Storybook/ExternalLayout.stories.tsx:12]: I think that import didn't work here so I had to use require. This is also the case in all other stories written for SLV and WLV.

File: typescript/packages/well-log-viewer/src/Storybook/helpers/mui-components.tsx (hunk 1 above)
Comment (mirisb) [reply typescript/packages/well-log-viewer/src/Storybook/helpers/mui-components.tsx:24]: Unfortunately I couldn't get rid of tooltip - without it the tabs component collapses. I did however remove export key words in that file that I don't use in my example.

File: typescript/packages/well-log-viewer/src/Storybook/ExternalLayout.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+                    <SyncLogViewer {...{ ...args, id: "c1" }} />
+                </div>
+            </div>
+        </Grid>
+    );
+};
+
+// storybook page
+const meta: Meta<typeof RTCWellLogViewer> = {
+    title: "WellLogViewer / Viz Examples / classification layout",
```
[the remark this answers is not in the document]
Comment (mirisb) [reply typescript/packages/well-log-viewer/src/Storybook/ExternalLayout.stories.tsx]: ok. removed the "Viz" from the folder name

**Pull request merged**